test/spanner.ts

```diff
@@ -2579,6 +2579,11 @@ describe('Spanner with mock server', () => {
     it('should list databases', async () => {
       const [databases] = await instance.getDatabases();
       assert.strictEqual(databases.length, 2);
+      // Assert that listing the databases does not cause a session pool to be
+      // initialized for the databases.
+      for (const db of databases) {
+        assert.strictEqual((db.pool_ as SessionPool).size, 0);
```

Comment:
Can we relocate an adaptation of this test to `test/instance.ts`, to confirm when the Database instance is created, the `{min: 0}` object is passed?

Reply:
I've added the assertion to one of the existing tests here: https://github.com/googleapis/nodejs-spanner/blob/7821ac5a5e1d02f9f64cef2747f97a6ce8fed4c1/test/instance.ts#L1038